include/gamepad/gamepad.hpp

```diff
@@ -90,9 +128,22 @@ class Gamepad {
          * @return std::string A unique listener name
          */
         static std::string unique_name();
-        static Button Gamepad::*button_to_ptr(pros::controller_digital_e_t button);
+        static Button Gamepad::* button_to_ptr(pros::controller_digital_e_t button);
```

Comment:
I'm amazed this is valid in the first place, C++ moment

```suggestion
        static Button* Gamepad::button_to_ptr(pros::controller_digital_e_t button);
```

Reply:
nevermind this is valid